apps/condo/domains/analytics/hooks/useDashboardFilters.tsx

```diff
@@ -82,9 +82,7 @@ type UseDateRangeFilterType = () => {
 export const useDateRangeFilter: UseDateRangeFilterType = () => {
     const [dateRange, setDateRange] = useState<[Dayjs, Dayjs]>([dayjs().subtract(6, 'days'), dayjs()])
 
-    const disabledDate = useCallback((currentDate) => {
-        return currentDate && currentDate < dayjs().startOf('year')
-    }, [])
+    const disabledDate = useCallback(() => false, [])
```

Comment:
Disabling aggregation limit is not a good idea

Reply:
Now all years are disabled off except the current and previous one